tests/test_models.py

```diff
@@ -2287,6 +2299,55 @@ def test_get_context_explicit_enterprise_customer(self, role_name, expected_cont
 
         assert expected_context == enterprise_role_assignment.get_context()
 
+    def test_get_assignments_many_assignments(self):
+        """
+        Tests that get_assignments orders context such that active enterprise uuids
+        are listed first for a given role.
+        """
+        alpha_customer = factories.EnterpriseCustomerFactory(uuid=UUID('aaaaaaaa-0000-0000-0000-000000000000'))
+        beta_customer = factories.EnterpriseCustomerFactory(uuid=UUID('bbbbbbbb-0000-0000-0000-000000000000'))
+        delta_customer = factories.EnterpriseCustomerFactory(uuid=UUID('dddddddd-0000-0000-0000-000000000000'))
+
+        test_user = factories.UserFactory(email='test@example.com')
+
+        # The order matters: because delta is most recently linked (and active), all other linked
+        # enterprise users will get active=False, leaving only delta as the active, linked enterprise.
+        # Remember that link_user() creates or updates an ECU record, which will automatically
+        # assign the enterprise_learner role via a Django signal.
+        self._link_user(test_user.email, alpha_customer)
+        self._link_user(test_user.email, beta_customer)
+        self._link_user(test_user.email, delta_customer)
+
+        # Create admin role assignments explicitly.
+        for customer in (alpha_customer, delta_customer):
+            SystemWideEnterpriseUserRoleAssignment.objects.get_or_create(
+                user=test_user, role=roles_api.admin_role(),
+                enterprise_customer=customer, applies_to_all_contexts=False,
+            )
+
+        SystemWideEnterpriseUserRoleAssignment.objects.get_or_create(
+            user=test_user,
+            role=roles_api.openedx_operator_role(),
+            applies_to_all_contexts=True,
+        )
+
+        expected_assignments = [
+            (
+                ENTERPRISE_ADMIN_ROLE, [
+                    str(delta_customer.uuid), str(alpha_customer.uuid),
+                ],
+            ),
```

Comment:
[clarification] Is the `ENTERPRISE_ADMIN_ROLE` first here because it happens to come before `ENTERPRISE_LEARNER_ROLE` alphabetically? If we by chance have another role in the future that comes before `ENTERPRISE_ADMIN_ROLE` alphabetically, would it then come first? I'm wondering if the sort order of the roles should be more deterministic (e.g., consumers might expect active, `enterprise_learner` roles to always come first)?